src/middlewared/middlewared/api/base/excluded.py

```diff
@@ -9,7 +9,7 @@
 __all__ = ["Excluded", "excluded_field"]
 
 
-class ExcludedField(Any):
+class ExcludedField:
```

Comment:
Attempting to subclass `typing.Any` throws a `TypeError`. I do not believe this needs to derive from anything in order to do its job, but correct me if I'm wrong.

Reply:
Unit tests pass, it seems to be working fine.

Reply:
Just double checked and you're right, I was using Python 3.10. Python 3.11 does not raise a TypeError.